corelib/src/libs/SireMove/openmmpmefep.cpp

```diff
@@ -993,7 +1058,10 @@ void OpenMMPMEFEP::initialise(bool fullPME)
     // Molecule solutemol = solute.moleculeAt(0).molecule();
     int nions = 0;
 
-    QVector<bool> perturbed_energies_tmp{false, false, false, false, false, false, false, false, false};
+    QVector<bool> perturbed_energies_tmp(10);
```

Comment:
Construct with perturbed_energies_tmp(10, false)?

Reply:
That would have also worked, but I prefer to keep it as aligned as https://github.com/OpenBioSim/sire/commit/43ed4bc57fa9f407e13621de1420550c154131c9 as possible.